cluster-autoscaler/main.go

```diff
@@ -110,6 +110,7 @@ func multiStringFlag(name string, usage string) *MultiStringFlag {
 }
 
 var (
+	resourceName            = flag.String("lease-resource-name", "cluster-autoscaler", "The name of the cluster-autoscaler kubernetes resource for lease locking")
```

Comment:
Please change variable name to match the flag - `leaseResourceName`.

Nit regarding style: description could probably be simplified to sth like "The lease resource to use in leader election."

Reply:
Updated accordingly